src/components/ComputeModal.js

```diff
@@ -1015,9 +993,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       ]),
       div({ style: { gridColumnEnd: 'span 6', marginTop: '1.5rem' } }, [
         h(LabeledCheckbox, {
-          checked: computeConfig.autopause,
+          checked: computeConfig.autopauseThreshold !== 0,
```

Comment:
It'd be good to encapsulate the logic used to decide if autopause is enabled via something like

```javascript
const isAutopauseEnabled = threshold => threshold > 0
```

and then we can call it above and below.

Reply:
I like that, I'll make that change